ui/app/src/utils/content.ts

```diff
@@ -351,6 +366,7 @@ export function parseContentXML(
 		);
 		current.craftercms.dateCreated = getInnerHtml(doc.querySelector(':scope > createdDate_dt'));
 		current.craftercms.dateModified = getInnerHtml(doc.querySelector(':scope > lastModifiedDate_dt'));
+		current.craftercms.disabled = getInnerHtml(doc.querySelector(':scope > disabled'), { trim: true }) === 'true';
```

Comment:
Should use XmlKeys enum for these selectors

Reply:
There is no entry for `disabled` in XmlKeys (I guess because it's the same key). Should I leave it as it is right now?